src/Evolution/Executables/ScalarAdvection/EvolveScalarAdvection.hpp

```diff
@@ -157,7 +144,7 @@ struct EvolutionMetavars {
                     volume_dim, Tags::Time,
                     tmpl::push_back<observe_fields,
                                     ScalarAdvection::Tags::VelocityField<Dim>>,
-                    analytic_solution_fields>,
+                    observe_fields>,
```

Comment:
This line seems to be reviewed -- should we place `::Tags::AnalyticSolutionsOptional<observe_fields>` instead..?